docs/source/install-tutorial.rst

```diff
@@ -3,21 +3,36 @@
 Installation tutorial with Debian, NGinx and gunicorn
 =====================================================
 
-preliminary packages:
+Following this tutorial you'll have a running Debian+Nginx+gunicorn+bepasty (HTTP) instance using Python3 on a minimally installed Debian server.  It is strongly recommended to use HTTPS for Internet facing services - configuring Nginx for HTTPS is left to the reader.
+
+After installing a minimal Debian 10 server.
+
+Install preliminary required packages:
 
 ::
 
-  apt-get install build-essential nginx supervisor python-dev git-core python-pip python-virtualenv
+  apt-get install git-core python3-dev python3-pip virtualenv python3-virtualenv nginx supervisor 
 
 
-commands to run
+Some commands need to be run as root - others as the user bepasty.
+
+Commands to run as root:
 
 ::
 
   # add user bepasty to system
-  adduser bepasty
+  adduser bepasty --disabled-login
+  
+Commands to run as user bepasty:
+
+::
+
   # change to user bepasty
-  sudo su - bepasty
+  su - bepasty
+  # add path to bepasty/.bashrc
+  echo "export PATH=\"/home/bepasty/.local/bin:\$PATH\"" >> /home/bepasty/.bashrc
```

Comment:
use ~/.bashrc here also, see below.

Reply:
The above was added to suppress errors from when installing flask &  pygmentize during the pip3 install.

Reply:
what I meant is that in line 35, you use `~/.bashrc` but in line 33 you use `>> /home/bepasty/.bashrc` to refer to the same file.

i suggest to use `>> ~/.bashrc` in line 33 also.

Reply:
still wondering though: what exactly ends up being in `.local/bin` when following these instructions?

shouldn't all stuff end up in `./bin` of the virtual env we created and activated?